backend/src/main/java/gov/cdc/usds/simplereport/service/PatientLinkService.java

```diff
@@ -106,4 +106,19 @@ public PatientLink createPatientLink(UUID testOrderUuid) {
     PatientLink pl = new PatientLink(to);
     return plrepo.save(pl);
   }
+
+  public String getObfuscatedPatientNameFromLink(UUID internalId)
```

Comment:
also you can put this back in the original method `getObfuscatedPatientNameFromLink`, since there isnt much there

Reply:
Drat, and here I thought this was the Spring-like way to structure this 😂 moved in [8a6241f](https://github.com/CDCgov/prime-simplereport/pull/3305/commits/8a6241fff848bc22b585e79227bcb281f1871a2f)